File: Suuretu02.c
#include <stdio.h>

int main() {
    // Input n and m
    int n, m;
    printf("Enter n and m separated by a space: ");
    scanf("%d %d", &n, &m);

    // Input n numbers
    int numbers[n];
    printf("Enter n numbers separated by spaces: ");
    for (int i = 0; i < n; i++) {
        scanf("%d", &numbers[i]);
    }

    // Output the mth numerical value
    if (m >= 1 && m <= n) {
        printf("The %dth numerical value is: %d\n", m, numbers[m - 1]);
    } else {
        printf("Invalid input for m.\n");
    }

    return 0;
}

File: Suuretu02.py
def main():
    # Input n and m
    n, m = map(int, input("Enter n and m separated by a space: ").split())

    # Input n numbers
    numbers = list(map(int, input("Enter n numbers separated by spaces: ").split()))

    # Output the mth numerical value
    if 1 <= m <= n:
        print(f"The {m}th numerical value is: {numbers[m - 1]}")
    else:
        print("Invalid input for m.")


if __name__ == "__main__":
    main()

